src/niswitch/examples/niswitch_connectchannels.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
```

Comment:
I think file should be niswitch_connect_channels.py not niswitch_connectchannels.py

Reply:
done